spec.html

```diff
@@ -11097,7 +11196,8 @@ <h2>Syntax</h2>
 
         DecimalBigIntegerLiteral ::
           `0` BigIntLiteralSuffix
-          NonZeroDigit DecimalDigits? BigIntLiteralSuffix
+          NonZeroDigit
+          NonZeroDigit NumericLiteralSeparator? DecimalDigits BigIntLiteralSuffix
```

Comment:
```suggestion
          NonZeroDigit DecimalDigits? BigIntLiteralSuffix
          NonZeroDigit NumericLiteralSeparator? DecimalDigits BigIntLiteralSuffix
```

Reply:
I updated my suggestion to consider that if NLS is present, it needs to be followed up by mandatory DecimalDigits and BigIntLiteralSuffix. (I removed the `?` from DecimalDigits).

Without the NLS, the first line already covers everything well.